### PR TITLE
feat(html): add github ribbon cdn link

### PR DESCRIPTION
**Pull request by csnktms**
Adding the missing ribbons to the .html files.

### CODE REVIEW

File: 2.6/index.html [hunk 1]
```diff
@@ -128,6 +132,7 @@ <h2 id="thanks">Thanks</h2>
         </div>
         <footer>
             <p>Code on <a href="https://github.com/hugovk/drop-python">GitHub</a>.</p>
+            <a class="github-fork-ribbon" href="https://github.com/hugovk" data-ribbon="Fork me on GitHub" title="Fork me on GitHub">Fork me on GitHub</a>
```
Comment (hugovk) [inline 2.6/index.html]: Please could you update these to link to the repo rather then my user page?

https://github.com/hugovk/drop-python/

Comment (csnktms) [reply 2.6/index.html]: fixed!

**Comment by hugovk** [comment]
Thanks for this!

In the end two PRs were made, and I've merged #8 because it had an additional CSS fix:


image

->

image

But I appreciate your work and you should still get Hacktoberfest credit, thanks again!

**Comment by yuriik333** [comment]
Hello red yea

**Comment by sonkatomi** [comment]
wow, 5yrs passed!

**Pull request closed without merging**